src/promptflow/promptflow/_telemetry/activity.py

```diff
@@ -51,13 +52,15 @@ def log_activity(
     :type custom_dimensions: dict
     :return: None
     """
+    custom_dimensions = custom_dimensions or {}
+    context = OperationContext.get_instance()
+    request_id = custom_dimensions.get('request_id', context.get('request_id', str(uuid.uuid4())))
```

Comment:
why do you make this change?

Reply:
I want the request id to be consistent throughout the entire process, so I modified this.
But the request id is recorded in **last_log_activity_info**, I need to make some modifications in the information section.

Reply:
It might not work in SDK case, as it might be a long lasting process taking many operations

Reply:
I have already communicated with @D-W- . Let's do this first. If we need to modify the request id, we can directly use modify it through content ['request_id ']=xxx at  external function.